shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/main/java/org/apache/shardingsphere/sharding/algorithm/sharding/inline/InlineShardingAlgorithm.java

```diff
@@ -84,6 +85,14 @@ public String doSharding(final Collection<String> availableTargetNames, final Pr
         return result;
     }
     
+    private String getTargetShardingNode(final Closure<?> closure, final String columnName) {
+        try {
+            return closure.call().toString();
```

Comment:
@zjcnb Maybe we should consider `MissingMethodException` together.

Reply:
Yes